components/bsx/Offer/OfferTable.vue

```diff
@@ -1,114 +1,123 @@
 <template>
-  <b-table :data="offers">
-    <b-table-column
-      v-if="isBsxStats"
-      cell-class="is-vcentered is-narrow"
-      :label="$t('offer.collection')"
-      v-slot="props"
-      sortable>
-      <nuxt-link :to="`/bsx/collection/${props.row.nft.collection.id}`">
-        <p
-          class="limit-width-text"
-          :title="
-            props.row.nft.collection.name
-              ? props.row.nft.collection.name
-              : props.row.nft.collection.id
-          ">
-          {{
-            props.row.nft.collection.name
-              ? props.row.nft.collection.name
-              : props.row.nft.collection.id
-          }}
-        </p>
-      </nuxt-link>
-    </b-table-column>
-    <b-table-column
-      v-if="isBsxStats"
-      cell-class="is-vcentered is-narrow"
-      :label="$t('offer.nftName')"
-      v-slot="props"
-      sortable>
-      <nuxt-link :to="`/bsx/gallery/${props.row.nft.id}`">
-        <p
-          class="limit-width-text"
-          :title="props.row.nft.name ? props.row.nft.name : props.row.nft.id">
-          {{ props.row.nft.name ? props.row.nft.name : props.row.nft.id }}
-        </p>
-      </nuxt-link>
-    </b-table-column>
-    <b-table-column
-      cell-class="is-vcentered is-narrow"
-      field="caller"
-      :label="$t('offer.caller')"
-      v-slot="props"
-      sortable>
-      <nuxt-link :to="{ name: 'bsx-u-id', params: { id: props.row.caller } }">
-        <Identity :address="props.row.caller" inline noOverflow />
-      </nuxt-link>
-    </b-table-column>
+  <div>
+    <Pagination
+      :total="total"
+      :perPage="itemsPerPage"
+      v-model="currentPage"
+      replace
+      enableListenKeyboardEvent
+      preserveScroll />
+    <b-table :data="showList">
+      <b-table-column
+        v-if="isBsxStats"
+        cell-class="is-vcentered is-narrow"
+        :label="$t('offer.collection')"
+        v-slot="props"
+        sortable>
+        <nuxt-link :to="`collection/${props.row.nft.collection.id}`">
+          <p
+            class="limit-width-text"
+            :title="
+              props.row.nft.collection.name
+                ? props.row.nft.collection.name
+                : props.row.nft.collection.id
+            ">
+            {{
+              props.row.nft.collection.name
+                ? props.row.nft.collection.name
+                : props.row.nft.collection.id
+            }}
+          </p>
+        </nuxt-link>
+      </b-table-column>
+      <b-table-column
+        v-if="isBsxStats"
+        cell-class="is-vcentered is-narrow"
+        :label="$t('offer.nftName')"
+        v-slot="props"
+        sortable>
+        <nuxt-link :to="`gallery/${props.row.nft.id}`">
```

Comment:
```suggestion
        <nuxt-link :to="`gallery/${props.row.nft.id}`">
```
      <nuxt-link :to="`/bsx/gallery/${props.row.nft.id}`">

Reply:
/bsx/gallery

Reply:
@Jarsen136 I did change it, if you look at the file its updated

Reply:
no, I see it lack of '/' at the beginning

Reply:
oh oops, added now